cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CurrentLimitsMapping.java

```diff
@@ -18,16 +18,24 @@
 public class CurrentLimitsMapping {
 
     private final Map<String, CurrentLimitsAdder> adders = new HashMap<>();
+    private final Context context;
 
-    CurrentLimitsMapping() {
+    CurrentLimitsMapping(Context context) {
+        this.context = context;
     }
 
     public CurrentLimitsAdder getCurrentLimitsAdder(String id, Supplier<CurrentLimitsAdder> supplier) {
         return adders.computeIfAbsent(id, s -> supplier.get());
     }
 
     void addAll() {
-        adders.values().forEach(CurrentLimitsAdder::add);
+        adders.forEach((key, value) -> {
+            if (Double.isNaN(value.getPermanentLimit())) {
+                context.ignored(String.format("Operational Limit Set of %s", key), "An operational limit set must at least contain one value for permanent limit.");
```

Comment:
It’s weird to mix context information and logging. I don’t remember how it’s implemented but we should introduce a new interface for logging purpose and the Context class could inherit from this new interface to avoid breaking change.

Reply:
I do not see any design issue with this, logging is part of the context.
Anyway it is not the purpose of the PR.